gradle.properties

```diff
@@ -23,4 +23,4 @@ android.defaults.buildfeatures.buildconfig=true
 android.nonTransitiveRClass=false
 android.nonFinalResIds=false
 # Library version
-VERSION_NAME=3.2.1
+VERSION_NAME=4.0.0
```

Comment:
I think there's some sort of duplication of this; I think we use the lib version inside the maven-push file instead of this one; can I ask you to check where this is used?

Reply:
ignore me